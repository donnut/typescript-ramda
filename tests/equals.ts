import R_equals = require('../ramda/dist/src/equals');

declare const string: string;

// @dts-jest:pass:snap
R_equals(string);
// @dts-jest:pass:snap
R_equals(string)(string);
// @dts-jest:pass:snap
R_equals(string, string);

// @dts-jest:pass:snap
R_equals(undefined);
// @dts-jest:pass:snap
R_equals(null);
