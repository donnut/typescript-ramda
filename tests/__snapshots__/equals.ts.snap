// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`R_equals(null) 1`] = `"<T>(b: T | null) => boolean"`;

exports[`R_equals(string) 1`] = `"(b: string) => boolean"`;

exports[`R_equals(string)(string) 1`] = `"boolean"`;

exports[`R_equals(string, string) 1`] = `"boolean"`;

exports[`R_equals(undefined) 1`] = `"<T>(b: T | undefined) => boolean"`;
